src/screens/InputScreen.js

```diff
@@ -0,0 +1,51 @@
+import React, {Component} from 'react';
+import {SafeAreaView, StyleSheet} from 'react-native';
+
+import NavigationHeader from '../components/NavigationHeader';
+import {Input, Icon, Text} from 'react-native-elements';
+
+class InputScreen extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      name: 'Unknown',
+    };
+  }
+
+  render() {
+    return (
+      <>
+        <NavigationHeader title="Input" navigation={this.props.navigation} />
+        <SafeAreaView style={styles.view}>
+          <Text
+            h3
+            testID="name-display-testID"
+            accessibilityLabel="name-display-accessibilityLabel">
+            {this.state.name}
+          </Text>
+          <Input
+            label="Your name"
+            placeholder="Enter your name..."
+            leftIcon={<Icon name="account-box" color="#464547" />}
```

Comment:
what does this color mean?

Reply:
it's a color of the icon

Reply:
https://material.io/resources/icons/?icon=account_box&style=baseline

Reply:
I meant `#464547`, it's usually considered as bad design, since it's not clear what actual color is